src/Microsoft.ML.Data/Transforms/SelectColumnsTransform.cs

```diff
@@ -179,14 +179,14 @@ public sealed class Arguments : TransformInputBase
             public string[] DropColumns;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Specifies whether to keep or remove hidden columns.", ShortName = "hidden", SortOrder = 3)]
-            public bool KeepHidden = true;
+            public bool KeepHidden = false;
```

Comment:
>false [](start = 37, length = 5)

nit: We usually have Defaults class with const properties, which ease a bit spreading of parameters across code.